content/applications/sales/sales/amazon_connector/manage.rst

```diff
@@ -56,76 +67,94 @@ turn, notify the customer that the order (or a part of it) is on its way.
 Follow deliveries in FBA
 ========================
 
-When a **FBA** (Fulfilled by Amazon) order is synchronized in Odoo, a stock move is recorded for
-each sales order item so that it is saved in your system. Inventory managers can find such moves
-in :menuselection:`Inventory --> Reporting --> Product Moves`. They pick up products in a specific
-inventory location called **Amazon**. This location represents your stock in Amazon's warehouses
-and allows you to manage the stock of your products under the FBA program.
+When an *FBA* (Fulfilled by Amazon) order is synchronized in Odoo, a stock move is recorded for each
+sales order item. That way, it's saved in the system.
 
-.. tip::
-   To follow your Amazon (FBA) stock in Odoo, you can make an inventory adjustment after
-   replenishing it. You can also trigger an automated replenishment from reordering rules on the
-   Amazon location.
+Inventory managers can access these stock moves by navigating to :menuselection:`Inventory app -->
+Reporting --> Moves History`.
+
+These managers pick up products in a specific inventory location called, *Amazon*. This specific
+location represents stock in Amazon's warehouses, and allows users to manage the stock of products
+under the *FBA* program.
```

Comment:
@ksc-odoo I don't understand what you are trying to say here.
For FBA orders, the stock move is made automatically by the connector thanks to the shipping status of Amazon.

When sending new products to amazon, the user should create a picking (delivery order) to transfer these products from his warehouse to the amazon location.

Reply:
Okay. I replaced this unclear section with the suggestion you left. Feel free to give it another look to see if everything is making sense now. Again, I apologize for any "confusing" portions of these Amazon connector docs. Without proper access to a valid Amazon Marketplace account, I am unable to have hands-on experience working with it, and can only do my best to enhance the previously written documentation about it, which is why I'm hoping your brilliant expertise can help "fill in the gaps" and make things clear and accurate. Any and all help/guidance you can give me in this regard would be greatly appreciated. Thanks! @dalaOdoo